README.md

```diff
@@ -1,14 +1,13 @@
 # Hazel [![License](https://img.shields.io/github/license/TheCherno/Hazel.svg)](https://github.com/TheCherno/Hazel/blob/master/LICENSE)
 
-![Hazel](/Resources/Branding/Hazel_Logo_Text_Light_Square.png?raw=true "Hazel")
+![Hazel](/Resources/Branding/Hazel_Logo_Text_Light_Square.png?raw=true "Hazel Game Engine")
 
 Hazel is primarily an early-stage interactive application and rendering engine for Windows. Currently not much is implemented, however (almost) everything inside this repository is being created within YouTube videos, found at [thecherno.com/engine](https://thecherno.com/engine). 
 
 ## Getting Started
 Visual Studio 2017 or 2019 is recommended, Hazel is officially untested on other development environments whilst we focus on a Windows build.
 
-Start by cloning the repository with `git clone --recursive https://github.com/TheCherno/Hazel`.
-
+Start by cloning the repository with `git clone --recursive https://github.com/TheCherno/Hazel`. <br />
```

Comment:
HTML markup is not part of the Git Markdown syntax.

Reply:
```suggestion
Start by cloning the repository with `git clone --recursive https://github.com/TheCherno/Hazel`.

```